app/js/controllers.js

```diff
@@ -4415,7 +4416,7 @@ angular.module('myApp.controllers', ['myApp.i18n'])
       })
     }
 
-    Storage.get('notify_nodesktop', 'send_ctrlenter', 'notify_volume', 'notify_novibrate', 'notify_nopreview', 'notify_nopush').then(function (settings) {
+    Storage.get('notify_nodesktop', 'send_ctrlenter', 'notify_volume', 'notify_novibrate', 'notify_nopreview', 'notify_nopush', 'theme_darkmode').then(function (settings) {
```

Comment:
Perhaps this should be named just `theme`, or `active_theme`; that way it is more generic, (for further theming later on).